src/Components/Company/AddNewCompany/addNewCompanyComponent.tsx

```diff
@@ -53,6 +57,7 @@ export const AddNewCompanyComponent = (props: any) => {
   const [countries, setCountries] = useState<[]>([]);
   const [loadingList, setLoadingList] = useState<boolean>(false);
   const [regionsList, setRegionsList] = useState<any[]>([]);
+  const [companyRoleType, setCompanyRoleType] = useState<any>("");
```

Comment:
Cant we use the existing companyRole property instated of creating a state variable